### DISCUSSION TITLE
【Random Play】旧文明DVD影像记录精选—黄泉角色PV—「你的颜色」

### DISCUSSION THREAD

**Discussion by D111Z41SY** [share121/inter-knot#3197] [其他]
https://www.bilibili.com/video/BV1ndeMeTEiU/
来自画师【渣z先生】的分享，使用旧文明光盘介质记录和旧式电视播放。
![QQ截图20240820000058](https://github.com/user-attachments/assets/f27a6634-d04f-4ac7-94a9-8c9372cf20df)

**Comment by Kamisato100** [comment]
甚至还是索尼<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
